.github/workflows/ci.yaml

```diff
@@ -0,0 +1,112 @@
+name: CI
+
+on: [push, pull_request]
+
+env:
+  UNITY_VERSION: 2020.2.6
```

Comment:
Version will need to align at least with: https://github.com/getsentry/sentry-unity/blob/d30c7765172972162db86bac1ab4414a6aad75bf/src/Directory.Build.props#L5

Likely we'll have a build matrix with more versions though, but for now we should at least test against `2019.4.18f1`

Reply:
https://github.com/getsentry/sentry-unity/blob/d30c7765172972162db86bac1ab4414a6aad75bf/samples/unity-of-bugs/ProjectSettings/ProjectVersion.txt#L1-L2

Would be nice to control it in 1 place if possible.